system3d_info/src/console_visitor.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+#include <string>
+
+#include "system3d_info/console_visitor.h"
+#include "system3d_info/graphic_container.h"
+
+
+
+using namespace system3d_info;
+
+ConsoleVisitor::ConsoleVisitor()
+{
+}
+
+
+ConsoleVisitor::~ConsoleVisitor()
+{
+}
+
+void ConsoleVisitor::visit(GraphicContainer & parent_container)
```

Comment:
I don't see if GraphicContainer is changed in the method. Is so, it should be passed by constant reference

Reply:
this "visit" method inherit from parent class. Maybe, in future, one child class can have "visit" method and change the container ?